common/reactive-grpc-benchmarks/pom.xml

```diff
@@ -117,6 +117,31 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.0</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>8.29</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                <!-- Don't run checkstyle for this benchmark project as it seems to be trying to checkstyle
```

Comment:
I couldn't reproduce this behavior locally or in CI when running `mvn install` on the top-level pom.

Reply:
Not sure why it doesn't work as expected in my dev environment, but if it does in build/yours then feel free to include.